README.md

```diff
@@ -19,7 +19,18 @@ nodejs:
     ROOT_APPENDER: CNP
   configmap:
     VAR_A: VALUE_A
-    VAR_B: VALUE_B
+    VAR_B: VALUE_B  
+  keyVaults:
+    cmc:
+      excludeEnvironmentSuffix: false
```

Comment:
shouldn't need to be here, it's an advanced option

Reply:
ok .. np